extensions/nuget/src/components/package-detail.tsx

```diff
@@ -0,0 +1,79 @@
+import { Detail, environment } from "@raycast/api";
+import { useEffect, useState } from "react";
+import { searchReadMe } from "../nuget-client";
+import { NugetPackage } from "../NugetPackage";
+import { join } from "path";
+import { humanizeNumber } from "../utils";
+
+export interface PackageDetailProps {
+  package: NugetPackage;
+}
+
+interface State {
+  isReadmeLoading: boolean;
+  readme: string;
+}
+
+const nugetBaseUrl = "https://www.nuget.org/packages/";
+
+export default function PackageDetail(props: PackageDetailProps): JSX.Element {
+  const [state, setState] = useState<State>({ readme: props.package.description, isReadmeLoading: true });
+
+  useEffect(() => {
+    console.log("readme");
+    searchReadMe(props.package).then((res) => {
+      console.log(res);
+      setState((oldState) => ({ ...oldState, isReadmeLoading: false, readme: res }));
+    });
+  }, []);
+
+  function renderVersions(): JSX.Element[] {
+    let isFirst = true;
+
+    const _versions: JSX.Element[] = [];
+
+    const revs = [...props.package.versions].reverse();
+
+    revs.map((v) => {
+      _versions.push(
+        <Detail.Metadata.Link
+          key={v.version}
+          title={isFirst ? "Versions" : ""}
+          target={nugetBaseUrl + props.package.id + "/" + v.version}
+          text={v.version}
+        />
+      );
+      isFirst = false;
+    });
+
+    return _versions;
+  }
+
+  return (
+    <Detail
```

Comment:
Why not also have the copy to clipboard action available when viewing the details of the package?

Reply:
added